themes/default/static/programs/aws-lambda-stepfunctions-jsonhelper-csharp/aws-lambda-stepfunctions-jsonhelper-csharp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
We should actually keep this at 6, as it's still on the list of [supported versions](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle) (and we [officially support all supported versions](https://www.pulumi.com/docs/languages-sdks/dotnet/)). Is there functionality here that doesn't work in .NET 6?

Reply:
`make test` was failing for me when this was 6. i think its because i have 7 on mine instance

Reply:
Yeah I think if you install 6 locally, you'll be good. https://dotnet.microsoft.com/en-us/download/dotnet